CMakeLists.txt

```diff
@@ -148,7 +154,10 @@ set(SPIRV_SOURCES
 
 add_library(${SPIRV_TOOLS} ${SPIRV_SOURCES})
 spvtools_default_compile_options(${SPIRV_TOOLS})
-target_include_directories(${SPIRV_TOOLS} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
+target_include_directories(${SPIRV_TOOLS} PUBLIC
+  ${CMAKE_CURRENT_SOURCE_DIR}/include
+  ${CMAKE_CURRENT_BINARY_DIR})
+add_dependencies(${SPIRV_TOOLS} spirv-tools-build-version)
```

Comment:
Nit: this dependency should go on the executables, not the library.  I presume there's no way to ask the library for build-version?